CurrencyDotNet.RestClient/CurrencyRestClient.cs

```diff
@@ -162,5 +157,30 @@ public void UpdatePosition()
         {
             throw new NotImplementedException();
         }
+        /// <summary>
+        /// Returns tickers for all the symbols for the last 24 hours
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<CallResult<IEnumerable<Ticker>>> GetTickersAsync(
+            CancellationToken cancellationToken=default(CancellationToken))
+        {
+            var request = new GetTickerRequest(symbol: null);
+
+            return await _restApiProvider.GetRequestAsync<IEnumerable<Ticker>>(request,cancellationToken);
+        }
+        /// <summary>
+        /// Returns ticker for a selected symbol for the last 24 hours
+        /// </summary>
+        /// <param name="symbol"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<CallResult<Ticker>> GetTickerAsync(string symbol,
+            CancellationToken cancellationToken=default(CancellationToken))
+        {
+            var request=new GetTickerRequest(symbol: symbol);
```

Comment:
Your IDE is hungry and eating spaces, what's wrong with it?

Reply:
Fixed.